airbyte-db/lib/src/main/java/io/airbyte/db/instance/MinimumFlywayMigrationVersionCheck.java

```diff
@@ -31,17 +32,18 @@
    * @param timeoutMs
    */
   public static void assertDatabase(DatabaseInstance db, long timeoutMs) {
-    var currWaitingTime = 0;
+    var startTime = System.currentTimeMillis();
     var initialized = false;
     while (!initialized) {
-      if (currWaitingTime >= timeoutMs) {
+      if ((System.currentTimeMillis() - startTime) >= timeoutMs) {
         throw new RuntimeException("Timeout while connecting to the database..");
       }
 
       try {
         initialized = db.isInitialized();
```

Comment:
If `initialized` is `false`, it should also sleep a while, right? Otherwise, this method still calls the `isInitialized` method non-stop.

Reply:
yeap!